docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '2'
+
+services:
+  jekyll:
+    image: bretfisher/jekyll-serve
+    volumes:
+      - .:/site
+    ports:
+      - '4000:4000'
```

Comment:
Worth adding a newline to the end of the file?

Reply:
if you like? I don't hold a strong opinion either way on that convention but i'm happy to be consistent

Reply:
Not a biggie - my personal preference is always having them, so then it doesn't cause issues in diffs, but happy to leave them as-is